xrmocap/ops/triangulation/point_selection/reprojection_error_point_selector.py

```diff
@@ -0,0 +1,162 @@
+# yapf: disable
```

Comment:
We may need a test for this selector. See [test_camera_selector.py](https://github.com/openxrlab/xrmocap/blob/main/tests/ops/test_camera_selector.py) for example.